caffe2/python/operator_test/weighted_sum_test.py

```diff
@@ -4,20 +4,21 @@
 from __future__ import unicode_literals
 
 from caffe2.python import core
-from hypothesis import given
 import caffe2.python.hypothesis_test_util as hu
 import hypothesis.strategies as st
 import numpy as np
+import caffe2.python.serialized_test.serialized_test_util as serial
 
 
-class TestWeightedSumOp(hu.HypothesisTestCase):
+class TestWeightedSumOp(serial.SerializedTestCase):
 
-    @given(n=st.integers(5, 8), m=st.integers(1, 1),
-           d=st.integers(2, 4), grad_on_w=st.booleans(),
-           **hu.gcs_cpu_only)
+    @serial.given_and_seeded(
+        n=st.integers(5, 8), m=st.integers(1, 1), d=st.integers(2, 4),
+        grad_on_w=st.booleans(), **hu.gcs_cpu_only)
     def test_weighted_sum(self, n, m, d, grad_on_w, gc, dc):
         input_names = []
         input_vars = []
+        np.random.seed(2603)
```

Comment:
Can we add the seed as a parameter in serialize_with_seed (the annotation?)

Reply:
Sure thing